osu.Game/Screens/Play/PlayerSettings/AnalysisSettings.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Replays;
+using osu.Game.Rulesets.UI;
+
+namespace osu.Game.Screens.Play.PlayerSettings
+{
+    public abstract partial class AnalysisSettings : PlayerSettingsGroup
```

Comment:
This feels completely back to front. A settings class should not create the actual analyser class. Probably needs to be flipped? The `DrawableRuleset` should return a `ReplayAnalysis` class which can create the settings / analyser / whatever is required.